prow/github/client.go

```diff
@@ -795,7 +839,7 @@ func (c *Client) ReopenIssue(org, repo string, number int) error {
 		requestBody: map[string]string{"state": "open"},
 		exitCodes:   []int{200},
 	}, nil)
-	return err
+	return stateCannotBeChangedOrOriginalError(err)
```

Comment:
Instead of this, add the possible failing exit codes to the input to `c.request`, then handle them. See how it is done in `AssignIssue`.